contents/tutorials/bounce-rate.md

```diff
@@ -1,102 +1,136 @@
 ---
 title: How to calculate bounce rate
-date: 2023-06-27
+date: 2024-12-11
 author:
   - ian-vanagas
+  - bijan-boustani
 showTitle: true
 sidebar: Docs
 tags:
   - insights
   - hogql
   - product analytics
+  - web analytics
 ---
 
-Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+Bounce rate is the percentage of users who leave a page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors. This tutorial shows you how to calculate bounce rate in PostHog.
 
-This tutorial shows you how to calculate bounce rate in PostHog. To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js) and enable "record user sessions"  in [project settings](https://app.posthog.com/project/settings).
+To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js). And ensure the following settings are enabled in your project settings:
 
-> **How does Google Analytics 4 [calculate the bounce rate](https://support.google.com/analytics/answer/12195621?hl=en)?** It is the percentage of sessions that **do not** last longer than 10 seconds, have a conversion event, or have at least 2 pageviews or screenviews.
+- [Record user sessions](https://app.posthog.com/settings/environment#replay)
+- [Enable autocapture for web](https://app.posthog.com/settings/environment-autocapture#autocapture)
+
+## How do we define bounce rate?
+
+Your bounce rate is the percentage of sessions that resulted in a bounce. We define a bounce as a session where a visitor:
+
+- spent less than 10 seconds on a page
+- did not [autocapture](/docs/product-analytics/autocapture) any events
+- only had one pageview
```

Comment:
This seems to contradict our earlier defintion we mentioned in the opening paragraph

Reply:
I think it would work if we included this in the opening section: 

- Bounce rate is generally "go to page, do nothing, leave"
- Why bounce rate matters (maybe)
- For us, it is X. For GA, it is Y.
- To get started measuring bounce rate, set up PostHog